api/user.js

```diff
@@ -2,12 +2,37 @@ import { BAD_REQUEST, TYPE_JSON } from './defs.js'
 import { POST } from './router.js'
 import * as db from './db.js'
 
+// function to check the form's data
+const verify_informations = (body) => {
+   let checks = [];
+   for(let item in body){
```

Comment:
never use `for in` to iterate on object.
It is slow, and have a tricky behaviour (iterate over prototype)

Use one of `Object.values`, `Object.keys` or `Object.entries` according to your needs.

```js
for (const [name, item] of Object.entries(body)) {
 // code here
}

Reply:
ok